client/dns_client.mli

```diff
@@ -44,28 +37,32 @@ module type S = sig
   type t
   (** The abstract state of a DNS client. *)
 
-  val create : ?rng:(int -> Cstruct.t) -> ?nameserver:ns_addr -> stack -> t
-  (** [create ~rng ~nameserver stack] creates the state record of the DNS client. *)
+  val create : ?nameserver:ns_addr -> timeout:int64 -> stack -> t
+  (** [create ~nameserver ~timeout stack] creates the state record of the DNS client.
+      We use [timeout] (ns) as a time budget for connect and request timeouts. *)
```

Comment:
```suggestion
      We use [timeout] (nanoseconds) as a cumulative time budget for connect and request timeouts. *)
```
`ns` is an overloaded term (nameserver / nanoseconds), make explicit that this is a cumulative countdown.

Reply:
As above, we think `ns` as a unit is okay, we're gonna clarify the cumulative part, thanks.